src/main/kotlin/com/_404/cosmos/springpicnic/scenario/entity/Scenario.kt

```diff
@@ -0,0 +1,19 @@
+package com._404.cosmos.springpicnic.scenario.entity
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "scenario")
+open class Scenario(
+        @Id
+        @GeneratedValue(strategy = GenerationType.AUTO)
+        open var id: Long? = null,
```

Comment:
id 컬럼은 non null 일거같은데 그럼 ? 를 안 붙이는게 어떨까?
```suggestion
        open var id: Long,
```

Reply:
jpa entity class 가 no-arg 생성자를 가지고 있어야 하는데 id 를 디폴트로 지정할 수가 없어서 저렇게 했던 것인데 이건 더 알아보겠습니다.

Reply:
lazy init ? 키워드가 왠지 이럴 때 쓰는거같네